tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    private function rateLimitErrorResponse()
+    {
+        return array(
+            'error' => array(),
+        );
+    }
+
+    /**
+     * @expectedException Error\RateLimit
+     */
+    public function testRateLimit()
```

Comment:
If we're just passing the code through, I agree that we don't need a second test.